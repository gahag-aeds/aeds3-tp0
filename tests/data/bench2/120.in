120 60
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 
Sub 70 109
Sum 67 106
Sub 24 63
Min 69 108
Add 77 116
Sum 13 52
Add 3 42
Min 51 90
Add 69 108
Min 32 71
Add 53 92
Sum 38 77
Add 15 54
Max 65 104
Add 13 52
Sum 52 91
Sub 10 49
Max 60 99
Add 69 108
Sum 13 52
Sub 30 69
Max 60 99
Sub 68 107
Sum 38 77
Add 13 52
Min 71 110
Add 77 116
Max 18 57
Add 61 100
Sum 27 66
Sub 14 53
Max 2 41
Add 13 52
Sum 70 109
Sub 22 61
Min 41 80
Sub 2 41
Sum 24 63
Add 11 50
Min 22 61
Add 7 46
Min 16 55
Add 59 98
Min 21 60
Sub 80 119
Sum 71 110
Sub 24 63
Max 41 80
Add 3 42
Sum 13 52
Add 53 92
Sum 64 103
Sub 24 63
Min 40 79
Sub 22 61
Max 36 75
Sub 62 101
Min 50 89
Add 5 44
Max 42 81
