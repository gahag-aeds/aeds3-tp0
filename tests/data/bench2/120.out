2305
1
1795
1
0
1676
100
2001
100
1866
101
1663
2
96
1669
95
2387
0
1713
0
0
0
2441
97
1897
2399
-1
95
-2
97
